debian/jessie/foreman-installer/changelog

```diff
@@ -1,3 +1,9 @@
+foreman-installer (1.15.0~rc1-1) stable; urgency=low
+
+  * 1.15.0-RC1 released
+
+ -- Daniel Lobato Garcia <me@daniellobato.me>  Thu, 06 Apr 2017 13:50:56 +0200
+
 foreman-installer (1.15.0-1) stable; urgency=low
```

Comment:
This entry shouldn't be present, and the 1.14 entries are missing. I think 602c743 is missing from the branch, it should have been branched _after_ that commit.